### PR TITLE
update python api for blob support

### PR DESCRIPTION
**Pull request by sayanshaw24**

### CODE REVIEW

File: onnxruntime_extensions/cvt.py [hunk 1]
```diff
@@ -47,6 +204,9 @@ def gen_processing_models(processor: Union[str, object],
         Keyword arguments for generating the post-processing model
     opset: int
         the target opset version of the model
+    cached: bool
```
Comment (wenbingl) [inline onnxruntime_extensions/cvt.py:207]: can a better name for this argument? 'cached' look like being irrelevant to what we are doing here.

**Comment by wenbingl** [comment]
why are the CI pipelines skipped?

**Comment by sayanshaw24** [comment]
> why are the CI pipelines skipped?

not sure - let me try to manually run it.

**Comment by sayanshaw24** [comment]
> pipeline triggers exclude this branch

ok seems pipeline triggers exclude this branch so it cannot be run from it (we likely need a PR to main branch, if we check in your hfjst branch in soon in #859, I can pull the changes and make a separate PR to main)

**Pull request closed without merging**